tm2/pkg/sdk/package.go

```diff
@@ -15,4 +15,6 @@ var Package = amino.RegisterPackage(amino.NewPackage(
 	).
 	WithTypes(
 		Result{},
+		DetailedEvent{},
+		EventAttribute{},
```

Comment:
```suggestion
		EventDetail{},
```

I think we should rename to Detail here too.

FYI, it aligns with how gRPC named the versatile attributes: https://pkg.go.dev/google.golang.org/grpc/internal/status#Status.Details